### PR TITLE
AISDK-168: Add transcriber and start ts

### PR DESCRIPTION
**Pull request by Chanc3**

### CODE REVIEW

File: src/models/RevAiApiJob.ts [hunk 1]
```diff
@@ -14,4 +14,13 @@ export interface RevAiApiJob {
     media_url?: string;
     failure?: string;
     failure_detail?: string;
+    delete_after_seconds?: number;
+    skip_diarization?: boolean;
+    skip_punctuation?: boolean;
+    remove_disfluencies?: boolean;
+    filter_profanity?:boolean;
+    custom_vocabulary_id?:string;
```
Comment (k-weng) [inline src/models/RevAiApiJob.ts]:
```suggestion
    filter_profanity?: boolean;
    custom_vocabulary_id?: string;
```

File: test/unit/api-client/api-client.submit.spec.ts [hunk 2]
```diff
@@ -287,7 +293,8 @@ describe('api-client job submission', () => {
                 filter_profanity: null,
                 remove_disfluencies: null,
                 delete_after_seconds: null,
-		language: null
+		        language: null,
+                transcriber: null
```
Comment (k-weng) [inline test/unit/api-client/api-client.submit.spec.ts]:
```suggestion
	        language: null,
                transcriber: null
```

File: examples/async_transcribe_media_from_url.js [hunk 3]
```diff
@@ -22,7 +22,12 @@ const token = require('./config/config.json').access_token;
                 "vocabularies",
                 "here"
             ]
-        }]
+        }],
+        filter_profanity: false, // Optional value available with some languages
+        remove_disfluencies: false, // Optional value available with some languages
+        delete_after_seconds: 2592000, // 30 days in seconds
+        language: "en", // Supported ISO 639-1 (2-letter) or ISO 639-3 (3-letter) language code
+        transcriber: "machine_v2"
```
Comment (k-weng) [inline examples/async_transcribe_media_from_url.js]: Add a comment here

File: examples/async_transcribe_local_media_file.js [hunk 4]
```diff
@@ -28,7 +28,8 @@ const token = require('./config/config.json').access_token;
         filter_profanity: false, // Optional value available with some languages
         remove_disfluencies: false, // Optional value available with some languages
         delete_after_seconds: 2592000, // 30 days in seconds
-        language: "en" // Supported ISO 639-1 (2-letter) or ISO 639-3 (3-letter) language code
+        language: "en", // Supported ISO 639-1 (2-letter) or ISO 639-3 (3-letter) language code
+        transcriber: "machine_v2"
```
Comment (k-weng) [inline examples/async_transcribe_local_media_file.js]: Add a comment here

**Review by amikofalvy — CHANGES_REQUESTED** [review]
Changes look good, but we need to bump the version

**Comment by amikofalvy** [comment]
Also, we might want to add integration tests for the new streaming and async job parameters

File: examples/streaming_transcribe_from_local_file.js [hunk 5]
```diff
@@ -32,7 +32,10 @@ const sessionConfig = new revai.SessionConfig(
     "my example metadata",  /* (optional) metadata */
     null,  /* (optional) custom_vocabulary_id */
     false, /* (optional) filter_profanity */
-    false /* (optional) remove_disfluencies */
+    false, /* (optional) remove_disfluencies */
+    0, /* (optional) delete_after_seconds */
+    0, /* (optional) start_ts */
+    "machine_v2" /* (optional) transcriber */
```
Comment (kbridbur) [inline examples/streaming_transcribe_from_local_file.js]: This should not be the default. streaming v2 is in closed beta

**Review by kbridbur — CHANGES_REQUESTED** [review]
Streaming example should not use machine_v2

**Review by k-weng — CHANGES_REQUESTED** [review]

**Comment by k-weng** [comment]
> Changes look good, but we need to bump the version

@amikofalvy I think we can release all of this under `3.0.0` so no need to bump version

**Review by amikofalvy — APPROVED** [review]

**Review by k-weng — APPROVED** [review]

**Review by kbridbur — APPROVED** [review]

**Pull request merged**